lms/djangoapps/instructor/views/api.py

```diff
@@ -220,6 +220,15 @@ def wrapped(*args, **kwargs):  # pylint: disable=missing-docstring
     return decorator
 
 
+class IsCourseStaff(permissions.BasePermission):
```

Comment:
Could you move this to the same file that has the IsCourseStaffInstructor permission, and add a corresponding test?

(openedx/core/lib/api/permissions.py)